resourcecontrollerv2/resource_controller_v2_examples_test.go

```diff
@@ -733,11 +734,23 @@ var _ = Describe(`ResourceControllerV2 Examples Tests`, func() {
 			b, _ := json.MarshalIndent(reclamationsList, "", "  ")
 			fmt.Println(string(b))
 
+			listReclamationsOptionsRgID := resourceControllerService.NewListReclamationsOptions()
```

Comment:
It looks like we now have two invocations of the ListReclamations() operation within this example code fragment.  Let's limit it to just one invocation.  If you can't incorporate the new query param into the existing example code fragment (i.e. use both account id and resource group id when calling ListReclamations() ), then just pick which parameter to use in the example.

Reply:
Also, there was a gosec error during the travis build. You will see in your PR's diff that by re-generating the code you ended up removing a "gosec" comment pertaining to the offending line of code.  Just restore that comment, then you can run `make all` and then `make scan-gosec` in your sandbox to make sure everything is clean after that.

Reply:
Thank you for the review @padamstx. I have made the changes suggested by you